### ISSUE TITLE
Playwrite with typescript not working

### ISSUE THREAD

**Issue by JackDra** [sveltejs/kit#4478]
### Describe the bug

Basic sveltekit app with typescript and palywrite enabled is not working for me.

### Reproduction

Setting up a basic sveltekit app:
```bash
npm init svelte@next my-app
cd my-app
npm install
npm run dev -- --open
```

With all options enabled, I.e. typescript and playwrite enabled causes the tests not to work.

### Logs

```shell
$ npm run test

> my-app@0.0.1 test
> playwright test


Running 0 test using 0 worker


Error: tests\test.ts: Cannot import a typescript file from an esmodule.



=================
 no tests found.
=================

```

### System Info

```shell
I am on windows machine, I am not sure if this issues happens on other platforms but the error does not seem platform dependent


$ npx envinfo --system --binaries --browsers --npmPackages "{svelte,@sveltejs/*,vite}"
Need to install the following packages:
  envinfo
Ok to proceed? (y) y

  System:
    OS: Windows 10 10.0.19041
    CPU: (12) x64 Intel(R) Core(TM) i7-9850H CPU @ 2.60GHz  
    Memory: 17.81 GB / 31.71 GB
  Binaries:
    Node: 16.14.2 - C:\Program Files\nodejs\node.EXE        
    npm: 8.5.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (99.0.1150.55)
    Internet Explorer: 11.0.19041.1202
  npmPackages:
    @sveltejs/adapter-auto: next => 1.0.0-next.34 
    @sveltejs/kit: next => 1.0.0-next.304 
    svelte: ^3.46.0 => 3.46.6 
```

### Severity

serious, but I can work around it

### Additional Information

To work around this, and probably the way to solve the issue has 2 steps:

playwrite has an alpha release that fixes this issue, see: https://github.com/microsoft/playwright/issues/11755#issuecomment-1082544994

Steps for me were:
```bash
npm up @playwright/test@next
npm exec playwright install
```

And then there is also an issue where the default playwrite file `playwright.config.js` is made in javascript, and needs to be build as a typescript file if typescript is enabled ` playwright.config.ts`. So my workaround is just to change the extension of the filename to `.ts`

**Comment by DoisKoh** [comment]
I'm having the exact same issue. Switching to 1.20.0 of Playwright also circumvents this.

**Comment by JackDra** [comment]
Closed by #4479, as @Rich-Harris updated PR to use playwright `@next` version if using typescript.  
Tracking the `@playwright/test@next` removal in #4488
